servicebroker/lib/config/setting.js

```diff
@@ -14,16 +14,17 @@ module.exports = function(settingsObj) {
     }
   };
 
-  var apiServer = function(apiServerUri) {
+  var cleanupAPIServerURI = function(apiServerUri) {
```

Comment:
Let us be clear. This is not specific to the APIServer and should have been name as to what it does like removeLastSlash since it would do it for any URL.